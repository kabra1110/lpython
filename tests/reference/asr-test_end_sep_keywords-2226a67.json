{
    "basename": "asr-test_end_sep_keywords-2226a67",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/test_end_sep_keywords.py",
    "infile_hash": "5ea30711228d4ebb64266988c1a706a3d64f196457b939ed3bf15ecf",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-test_end_sep_keywords-2226a67.stdout",
    "stdout_hash": "3a586fde04b4d918381f4de704d1505e74617f283c1f33ba6aa8a538",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}