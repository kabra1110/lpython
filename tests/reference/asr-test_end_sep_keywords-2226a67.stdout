(TranslationUnit
    (SymbolTable
        1
        {
            __main__:
                (Module
                    (SymbolTable
                        2
                        {
                            __main____global_statements:
                                (Function
                                    (SymbolTable
                                        4
                                        {
                                            
                                        })
                                    __main____global_statements
                                    (FunctionType
                                        []
                                        ()
                                        Source
                                        Implementation
                                        ()
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        []
                                        .false.
                                    )
                                    [testEndSepKeywords]
                                    []
                                    [(SubroutineCall
                                        2 testEndSepKeywords
                                        ()
                                        []
                                        ()
                                    )]
                                    ()
                                    Public
                                    .false.
                                    .false.
                                    ()
                                ),
                            testEndSepKeywords:
                                (Function
                                    (SymbolTable
                                        3
                                        {
                                            
                                        })
                                    testEndSepKeywords
                                    (FunctionType
                                        []
                                        ()
                                        Source
                                        Implementation
                                        ()
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        []
                                        .false.
                                    )
                                    []
                                    []
                                    [(Print
                                        ()
                                        [(StringConstant
                                            "abc"
                                            (Character 1 3 ())
                                        )
                                        (StringConstant
                                            "lmn"
                                            (Character 1 3 ())
                                        )
                                        (StringConstant
                                            "pqr"
                                            (Character 1 3 ())
                                        )]
                                        ()
                                        ()
                                    )
                                    (Print
                                        ()
                                        [(StringConstant
                                            "abc"
                                            (Character 1 3 ())
                                        )
                                        (StringConstant
                                            "lmn"
                                            (Character 1 3 ())
                                        )
                                        (StringConstant
                                            "pqr"
                                            (Character 1 3 ())
                                        )]
                                        (StringConstant
                                            "+"
                                            (Character 1 1 ())
                                        )
                                        ()
                                    )
                                    (Print
                                        ()
                                        [(StringConstant
                                            "abc"
                                            (Character 1 3 ())
                                        )
                                        (StringConstant
                                            "lmn"
                                            (Character 1 3 ())
                                        )
                                        (StringConstant
                                            "pqr"
                                            (Character 1 3 ())
                                        )]
                                        ()
                                        (StringConstant
                                            "xyz\n"
                                            (Character 1 4 ())
                                        )
                                    )
                                    (Print
                                        ()
                                        [(StringConstant
                                            "abc"
                                            (Character 1 3 ())
                                        )
                                        (StringConstant
                                            "lmn"
                                            (Character 1 3 ())
                                        )
                                        (StringConstant
                                            "pqr"
                                            (Character 1 3 ())
                                        )]
                                        (StringConstant
                                            "+"
                                            (Character 1 1 ())
                                        )
                                        (StringConstant
                                            "xyz\n"
                                            (Character 1 4 ())
                                        )
                                    )]
                                    ()
                                    Public
                                    .false.
                                    .false.
                                    ()
                                )
                        })
                    __main__
                    []
                    .false.
                    .false.
                ),
            main_program:
                (Program
                    (SymbolTable
                        5
                        {
                            __main____global_statements:
                                (ExternalSymbol
                                    5
                                    __main____global_statements
                                    2 __main____global_statements
                                    __main__
                                    []
                                    __main____global_statements
                                    Public
                                )
                        })
                    main_program
                    [__main__]
                    [(SubroutineCall
                        5 __main____global_statements
                        2 __main____global_statements
                        []
                        ()
                    )]
                )
        })
    []
)
